packages/tooltip/src/main.js

```diff
@@ -157,6 +146,18 @@ export default {
       this.debounceClose();
     },
     handleFocus() {
+      if (!this.$slots.default || !this.$slots.default.length) {
+        this.doFocus();
+        return;
+      }
+      const instance = this.$slots.default[0].componentInstance;
+      if (instance && instance.focus) {
+        instance.focus();
+      } else {
```

Comment:
I think we should only 'doFocus' if instance exists. I was considering putting in a PR for this exact issue, which is causing tooltips to show when certain elements re-render in my app. What do you think @lianzhao?
```suggestion
      } else if (instance) {
```

Reply:
@lianzhao can you take a look at this? We are experiencing the same issues.

Reply:
this issue haven't be resolved? too many detached! who can tell me will publish to which version?
![image](https://user-images.githubusercontent.com/54703532/155520236-6f40b8db-3ef4-4f8c-a6b4-d8aea613273f.png)